examples/app.py

```diff
@@ -27,18 +27,23 @@
 
 For this example the access control is disabled.
 
-Run example development server:
+Install requirements:
 
 .. code-block:: console
 
+    $ pip install -e .[all]
     $ cd examples
-    $ pip install -r requirements.txt
-    $ export FLASK_APP=app.py
-    $ flask db init
-    $ flask db create
-    $ flask index init
-    $ flask fixtures records
-    $ flask run --debugger
+    $ # clean the environment if needed
```

Comment:
I would just move it at the end of the docstring so people don't loose the focus.